test/storage/b_plus_tree_insert_test.cpp

```diff
@@ -39,6 +39,49 @@ TEST(BPlusTreeTests, DISABLED_InsertTest1) {
   auto header_page = bpm->NewPage(&page_id);
   (void)header_page;
 
+  int64_t key = 42;
+  int64_t value = key & 0xFFFFFFFF;
+  rid.Set(static_cast<int32_t>(key >> 32), value);
```

Comment:
`key >> 32` is zero? If so why do we set key = 42 and do some "transformations" instead of directly `int key = 0`?

Reply:
Good catch. This is an artifact from the previous test1 from which this was copied.